python/ray/_private/services.py

```diff
@@ -1037,12 +1037,7 @@ def start_dashboard(require_dashboard,
             raise ValueError(
                 f"The given dashboard port {port} is already in use")
 
-    if "RAY_USE_NEW_DASHBOARD" in os.environ:
-        dashboard_dir = "new_dashboard"
-    else:
-        dashboard_dir = "dashboard"
-        logdir = None
-
+    dashboard_dir = "new_dashboard"
```

Comment:
I assume we'll remove the "new" prefix in a future PR

Reply:
Yeah, that'll be done when we remove the old dashboard, which we decided not to do just yet in case reverting is needed.